### PR TITLE
Fix migrating db that isn't past squashed migration

### LINKED ISSUE #303: 1.3.7b3 migration fails if only partially migrated
Migration status:
```
easyaudit                                                                                                                                                                                                          
 [X] 0001_initial                                                                                                                                                                                                  
 [X] 0002_auto_20170125_0759                                                                                                                                                                                       
 [X] 0003_auto_20170228_1505                                                                                                                                                                                       
 [X] 0004_auto_20170620_1354                                                                                                                                                                                       
 [X] 0005_auto_20170713_1155                                                                                                                                                                                       
 [X] 0006_auto_20171018_1242
 [X] 0007_auto_20180105_0838
 [X] 0008_auto_20180220_1908
 [X] 0009_auto_20180314_2225
 [X] 0010_repr_text
 [X] 0011_auto_20181101_1339
 [X] 0012_auto_20181018_0012
 [X] 0013_auto_20190723_0126
 [X] 0014_auto_20200513_0008
 [X] 0015_auto_20201019_1217
 [X] 0016_alter_crudevent_event_type
 [X] 0017_alter_requestevent_datetime
 [ ] 0018_rename_crudevent_object_id_content_type_index
 [ ] 0019_alter_crudevent_changed_fields_and_more
```

Error observed:
```
Operations to perform:                                                                                                                                                                                             
  Apply all migrations: easyaudit                                              
Traceback (most recent call last):                                                                                                                                                                                 
  File "/var/www/manage.py", line 22, in <module>                                                                                                                                                                  
    execute_from_command_line(sys.argv)                                                                                                                                                                            
  File "/usr/local/lib/python3.11/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line                                                                                        
    utility.execute()                                                                                                                                                                                              
  File "/usr/local/lib/python3.11/site-packages/django/core/management/__init__.py", line 436, in execute                                                                                                          
    self.fetch_command(subcommand).run_from_argv(self.argv)                                                                                                                                                        
  File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 413, in run_from_argv                                                                                                        
    self.execute(*args, **cmd_options)                                                                                                                                                                             
  File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 459, in execute                                                                                                              
    output = self.handle(*args, **options)                                                                                                                                                                         
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                         
  File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 107, in wrapper                                                                                                              
    res = handle_func(*args, **kwargs)                                                                                                                                                                             
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                             
  File "/usr/local/lib/python3.11/site-packages/django/core/management/commands/migrate.py", line 303, in handle                                                                                                   
    pre_migrate_apps = pre_migrate_state.apps                                                                                                                                                                      
                       ^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                      
  File "/usr/local/lib/python3.11/site-packages/django/utils/functional.py", line 47, in __get__                                                                                                                   
    res = instance.__dict__[self.name] = self.func(instance)                                                                                                                                                       
                                         ^^^^^^^^^^^^^^^^^^^                                                                                                                                                       
  File "/usr/local/lib/python3.11/site-packages/django/db/migrations/state.py", line 565, in apps                                                                                                                  
    return StateApps(self.real_apps, self.models)                                                                                                                                                                  
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                  
  File "/usr/local/lib/python3.11/site-packages/django/db/migrations/state.py", line 626, in __init__                                                                                                              
    self.render_multiple([*models.values(), *self.real_models])                                                                                                                                                    
  File "/usr/local/lib/python3.11/site-packages/django/db/migrations/state.py", line 664, in render_multiple                                                                                                       
    model.render(self)                                                                                                                                                                                             
  File "/usr/local/lib/python3.11/site-packages/django/db/migrations/state.py", line 957, in render                                                                                                                
    return type(self.name, bases, body)                                                                                                                                                                            
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                            
  File "/usr/local/lib/python3.11/site-packages/django/db/models/base.py", line 143, in __new__                                                                                                                    
    new_class.add_to_class("_meta", Options(meta, app_label))                                                                                                                                                      
  File "/usr/local/lib/python3.11/site-packages/django/db/models/base.py", line 371, in add_to_class                                                                                                               
    value.contribute_to_class(cls, name)                                                                                                                                                                           
  File "/usr/local/lib/python3.11/site-packages/django/db/models/options.py", line 220, in contribute_to_class                                                                                                     
    raise TypeError(                                                                                                                                                                                               
TypeError: 'class Meta' got invalid attribute(s): index_together                                                           
```

### PR DESCRIPTION
**Pull request by PaarthShah**
Retroactively changing the migration is the only way for this to work: it's a safe operation on its own because:
1. The migration has already been run
2. the operation itself is identical (it was copied from the squashed migration)
3. for a fresh migration, only the squashed version is run anyway

Fixes #303

Migration state after:
```
easyaudit                                                                                                                                                                        
 [X] 0001_initial                                                                                                                                                                
 [X] 0002_auto_20170125_0759                                                                                                                                                     
 [X] 0003_auto_20170228_1505                                                                                                                                                     
 [X] 0004_auto_20170620_1354_squashed_0019_alter_crudevent_changed_fields_and_more (16 squashed migrations)
```

### CODE REVIEW

File: easyaudit/migrations/0004_auto_20170620_1354.py
Comment (jourdanrodrigues) [inline easyaudit/migrations/0004_auto_20170620_1354.py:16]: I feel like you either:

1. Shouldn't add the index name
2. Should remove the operation that introduces the index name, migration 0018

**Review by jourdanrodrigues — CHANGES_REQUESTED** [review]

File: easyaudit/migrations/0004_auto_20170620_1354.py
Comment (PaarthShah) [reply easyaudit/migrations/0004_auto_20170620_1354.py:16]: I'd be good with removing both; I just wanted it to be absolutely consistent. My bullet points in the PR description should hold true on why specifying it doesn't matter that much, though

Comment (jourdanrodrigues) [reply easyaudit/migrations/0004_auto_20170620_1354.py:16]: That's actually not both, you just need to pick one.

If you leave it as is, the migration 0018 will break because the index already exists.

This will break test suites that recreate the database from scratch, which is Django's default behavior.

Comment (PaarthShah) [reply easyaudit/migrations/0004_auto_20170620_1354.py]: Implemented suggestion 1

**Review by scribejourdan — APPROVED** [review]

**Comment by jourdanrodrigues** [comment]
My approval doesn't mean anything, sadly. 😬 Hope the maintainers show up soon enough.

File: easyaudit/migrations/0004_auto_20170620_1354.py
Comment (PaarthShah) [reply easyaudit/migrations/0004_auto_20170620_1354.py:16]: @jourdanrodrigues Turns out `name` is a required arg for `AddIndex`, so had to revert.

Comment (scribejourdan) [reply easyaudit/migrations/0004_auto_20170620_1354.py:16]: It makes sense, so we need to remove the operation 0018 (not the migration).

**Comment by jourdanrodrigues** [comment]
There's a need to run `poetry lock` @PaarthShah

**Comment by PaarthShah** [comment]
@jourdanrodrigues I'll get to that asap when I'm back at my computer

**Comment by mschoettle** [comment]
Could you bring your branch up to date with master? The lock file is updated and CI now runs with all supported combinations of Python and Django.

Ultimately, it is up to @jheld as the (sole?) maintainer to decide which approach to take (as discussed in #301).

**Comment by jheld** [comment]
@PaarthShah seems we need another rebase to get rid of the conflicts

**Comment by PaarthShah** [comment]
@jheld @mschoettle done

**Pull request merged**
